src/devices/Bmp180/CalibrationData.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Bmp180
+{
+    internal class CalibrationData
+    {
+        public short AC1 {get;set;}

```

Comment:
> AC1

Are these the exact names as they appear on the datasheet? If not, We should try renaming them to either match the datasheet, or to be more descriptive into what they are using PascalCasing.

Reply:
Yes, these are the exact names.


Page 13 -> https://cdn-shop.adafruit.com/datasheets/BST-BMP180-DS000-09.pdf

![bmp180-reg](https://user-images.githubusercontent.com/14133711/52100339-b860fa80-259c-11e9-94e4-b4e70096ba93.png)